odoo_updates/odoo_updates.py

```diff
@@ -266,6 +350,53 @@ def diff_to_screen(views_states, title):
                     click.secho(line)
 
 
+def fields_to_screen(fields_states, title):
+    show_model_field = dict()
+    for state, values in fields_states.iteritems():
+        click.secho('+ {state} {title}'.
+                    format(state=state.title(), title=title), fg='yellow')
+        for field in values:
+            if state == 'updated':
+                column = {'type': difflib.unified_diff(
+                        field['original'].get('type', '').split('\n'),
+                        field['modified'].get('type', '').split('\n')),
+                        'field_description': difflib.unified_diff(
+                            field['original'].get('field_description', '').split('\n'),
+                            field['modified'].get('field_description', '').split('\n')), }
+                field.update(column)
+            else:
+                field.update({'type': field['type'].split('\n'),
+                              'field_description': field['field_description'].split('\n'), })
```

Comment:
@mapuerta Here are you updating the field['type'] and field['field_description'] with the same values it already has?